### PR TITLE
Send SES Email metrics to CloudWatch

### PR DESCRIPTION
**Pull request by joehoyle**
See https://github.com/humanmade/hm-platform/pull/11/commits/d023f73e4456154459f5854aa49b6b8dc302dc59 for actual review code. The rest is a needed SDK update.

### CODE REVIEW

**Comment by rmccue** [comment]
FWIW, I hate that we have the SDK committed here.

**Comment by joehoyle** [comment]
Did you add that?

File: lib/ses-to-cloudwatch/plugin.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		if ( $streams ) {
+			$params['sequenceToken'] = $streams[0]['uploadSequenceToken'];
+		}
+		cloudwatch_logs_client()->putLogEventsAsync( $params );
+	} catch ( Exception $e ) {
+		trigger_error( 'Failed to send email to CloudWatch: ' . $e->getMessage(), E_USER_WARNING );
+	}
+}
+
+function cloudwatch_logs_client() {
```
Comment (rmccue) [inline lib/ses-to-cloudwatch/plugin.php]: Can we call this `get_client` or something? This name is a bit weird. Also, needs phpDoc.

**Comment by joehoyle** [comment]
Closing now as not needed

**Pull request closed without merging**